<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoFiltros</class>
 <widget class="QDialog" name="DialogoFiltros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pesquisar por filtros</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Filtros</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="cbNome">
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtNome"/>
   </item>
   <item>
    <widget class="QCheckBox" name="cbTel">
     <property name="text">
      <string>Telefone</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtTel"/>
   </item>
   <item>
    <widget class="QCheckBox" name="cbEmail">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtEmail"/>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>CANCELAR</string>
       </property>
       <property name="icon">
        <iconset resource="Arquivos.qrc">
         <normaloff>:/Icones/close.png</normaloff>:/Icones/close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOk">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="Arquivos.qrc">
         <normaloff>:/Icones/magnify.png</normaloff>:/Icones/magnify.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Arquivos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btCancelar</sender>
   <signal>clicked()</signal>
   <receiver>DialogoFiltros</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
